qmk_wrapper.h

```diff
@@ -34,6 +34,8 @@
 
 #define TAPPING_TERM 200
 
+#define MAX(x, y) (((x) > (y)) ? (x) : (y))
```

Comment:
this should be removed as it's unsafe against double eval. should use `st_max` from utils.h instead.

Reply:
Yep, added specifically for that preproc key_stack capacity calculation. No intention to use it in running code.